tests/test_discount.py

```diff
@@ -0,0 +1,31 @@
+from pywr.parameters import Parameter, load_parameter
+import datetime
+
+class DiscountFactorParameter(Parameter):
+    """Parameter that returns the current discount factor based on discount rate and a base year.
+
+    Parameters
+    ----------
+    discount_rate : discount rate fixed across planning period
```

Comment:
Doc should state whether this is a % (0 - 100) or factor (0 - 1). The code is using the latter which is fine.

Reply:
Thanks, I added that to the doc.